src/components/Icon/icons/index.tsx

```diff
@@ -197,6 +199,7 @@ const icons: Record<IconName, SVGComponent> = {
   iconLocations: IconLocations,
   iconMenu: IconMenu,
   iconMenuHorizontal: IconMenuHorizontal,
+  iconMenuVertical: IconMenuVertical,
```

Comment:
Isn't "menuVertical" the same as this new "iconMenuVertical" ?

Reply:
Ugh, yes I didn't realize we already had the asset because storybook alphabetization is weird.  Will close this PR.